docs/modules/parquet/formats/geoparquet.md

```diff
@@ -4,17 +4,32 @@
 &emsp;
 ![apache-logo](../../../images/logos/apache-logo.png)
 
-- *[`loaders.gl/parquet`](/docs/modules/parquet)*
-- *[geoparquet.org](https://geoparquet.org)*
+- _[`loaders.gl/parquet`](/docs/modules/parquet)_
+- _[geoparquet.org](https://geoparquet.org)_
 
-Geoparquet is a set of conventions for storing geospatial data in Parquet files. 
+Geoparquet is a set of conventions for storing geospatial data in Parquet files.
 
 Standardization is happening at [geoparquet.org](https://geoparquet.org).
 
 GeoParquet file is a Parquet file that additionally follows these conventions:
 
 - Geospatial metadata describing any geospatial columns is stored in the Parquet file's schema metadata (as stringified JSON).
-- Geometry columns are [WKB](/docs/modules/wkt/formats/wkb) encoded (additional encodings will likely be added)/
+- Geometry columns are [WKB](/docs/modules/wkt/formats/wkb) encoded (additional encodings will likely be added).
+
+## Supported features checklist
+
+| Type                                                                                         | Supported |
+| -------------------------------------------------------------------------------------------- | --------- |
+| Parse file metadata                                                                          | ✅        |
+| Geometry column encoding: WKB                                                                | ✅        |
+| Geometry column encoding: single-geometry type encodings based on the GeoArrow specification | ❌        |
+| "crs" column metadata: reorder vertices if set "counterclockwise"                            | ❌        |
+| "orientation" column metadata: reorder vertices if set "counterclockwise"                    | ❌        |
+| "covering" column metadata: per-row bounding boxes                                           | ❌        |
```

Comment:
Are the bounding boxes per row group? If per row, they can't really be stored in metadata?

Reply:
"covering" metadata describes an additional column that stores bounding boxes data.
> A bounding box column MUST be a Parquet group field with 4 child fields named xmin, xmax, ymin, and ymax representing the geometry's coordinate range.

https://github.com/opengeospatial/geoparquet/blob/main/format-specs/geoparquet.md#bounding-box-columns